docs/list.md

```diff
@@ -154,10 +155,32 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
 
 - **Swag**: Reusable Bottles, Stickers (possibly? Awaiting updated swag for 2022.)
 - **Requirements**:
-    - Submit 1 or more pull requests to qualify as a contributor. Check out the [Contributing Guidelines](https://github.com/meilisearch/.github/blob/main/Hacktoberfest_2022_contributors_guidelines.md){:target="\_blank"} for more details.
-- **How to sign up**: Join the [Slack community](https://www.notion.so/6cd1509802984122ab74745f637b694f){:target="\_blank"} and the [#hacktoberfest channel](https://meilicommunity.slack.com/archives/C02G1H43J2X){:target="\_blank"} to connect with Meilisearch employees and for any queries.
-- **Issues**: See the list of [hacktoberfest-labeled issues](https://github.com/search?q=org%3Ameilisearch+label%3Ahacktoberfest&state=open&type=Issues){:target="\_blank"}.
-- **Notes**: Check out the [blog post](https://blog.meilisearch.com/contribute-hacktoberfest-2022/){:target="\_blank"} for more info.
+<<<<<<< HEAD
+    - Submit 1 or more pull requests to qualify as a contributor. Check out the [Contributing Guidelines](https://github.com/meilisearch/.github/blob/main/Hacktoberfest_2022_contributors_guidelines.md) for more details.
+- **How to sign up**: Join the [Slack community](https://www.notion.so/6cd1509802984122ab74745f637b694f) and the [#hacktoberfest channel](https://meilicommunity.slack.com/archives/C02G1H43J2X) to connect with Meilisearch employees and for any queries.
+- **Issues**: See the list of [hacktoberfest-labeled issues](https://github.com/search?q=org%3Ameilisearch+label%3Ahacktoberfest&state=open&type=Issues).
+- **Notes**: Check out the [blog post](https://blog.meilisearch.com/contribute-hacktoberfest-2022/) for more info.
+>>>>>>> e2b3adf2093b5a410fba097c66210c512628e5e4
```

Comment:
It looks like you have a merge conflict here. Please remove the duplicate text and keep the ones that have `{:target="\_blank"}` after the links.